scripts/cargo-rmc

```diff
@@ -56,6 +41,12 @@ def main():
     if not rmc.dependencies_in_path():
```

Comment:
I know this isn't your change, but should this give a more helpful message?

Reply:
The error is printed by `dependencies_in_path`, so we don't need to print anything here.